opc/scripts/setup/wizard.py

```
@@ -605,29 +606,29 @@ async def run_setup_wizard() -> None:
     # Step 5: Generate .env
-    console.print("\n[bold]Step 5/13: Generating configuration...[/bold]")
+    console.print("\n[bold]Step 5/15: Generating configuration...[/bold]")
     config = {"database": db_config, "embeddings": embeddings, "api_keys": api_keys}
     env_path = Path.cwd() / ".env"
     generate_env_file(config, env_path)
     console.print(f"  [green]OK[/green] Generated {env_path}")
 
     # Step 5: Container stack (Sandbox Infrastructure)
     runtime = prereqs.get("container_runtime", "docker")
-    console.print(f"\n[bold]Step 6/13: Container Stack (Sandbox Infrastructure)[/bold]")
+    console.print(f"\n[bold]Step 6/15: Container Stack (Sandbox Infrastructure)[/bold]")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)